test/formatters/apheleia-ft.el

```diff
@@ -269,8 +269,10 @@ environment variable, defaulting to all formatters."
     (dolist (in-file (apheleia-ft--input-files formatter))
       (let ((extension (file-name-extension in-file))
             (in-text (apheleia-ft--read-file in-file))
+            ;; The `in-temp-real-file' variable is set to whatever
+            ;; temporary file the formatter will run on (in case it
+            ;; uses the `file' or `filepath' symbol or is a function).
```

Comment:
It took me a while to understand the purpose of this variable. Adding a comment to explain the difference from `in-temp-file`, which also doesn't actually have to be defined up here because it is only used temporarily and then discarded.